src/pages/CreateRecover/components/CreateRecoverPassword/components/CreateRecoverPasswodFormFields.tsx

```diff
@@ -68,7 +69,10 @@ export const CreateRecoverPasswordFormFields = ({
                 onVisibilityChange={toggleState}
               />
             )}
-            {showError(field.name) && (
+            {getFormHasError({
```

Comment:
This could be stored in a variable and have that used for conditionals, since this function is called at least three times in this file. 👍